site_cmp3/site_cmp3.sh

```diff
@@ -224,7 +225,19 @@ ed_action_errors=${out}/${RSE}_${now}_ED_action.errors
 $python actions/remove_empty_dirs.py -s $stats -c ${merged_config_file} -L 10000 $out $RSE 2> $ed_action_errors
 
 end_time=`date -u +%s`
+elapsed_time=$((end_time - timestamp))
 
 rce_update_stats $stats << _EOF_
-{ "end_time":${end_time}.0 }
+{ 
+    "end_time": ${end_time}.0,
+    "elapsed_time": ${elapsed_time}.0
+}
 _EOF_
+
+#
+# 7. Push2Prometheus
+#
+echo
+echo Pushing stats...
+echo
+$python /consistency/push2prometheus.py $stats site_cmp3
```

Comment:
Later, do you think errors from this should be piped to a directory as well or do you prefer to keep them in stdout?

Reply:
Subprocesses, triggered by this script, are called by redirecting their stderr to certain files. This file are placed under `/var/cache/consistency-dump` directory of the running container and can be used for debugging. The retention policy applied to these files is specified in [cleanup.sh](https://github.com/eachristgr/cms_consistency/blob/prometheus/docker/cleanup.sh).

Reply:
Does this call `site_cmp3`?